my-app/src/public-components/button/button.js

```diff
@@ -1,11 +1,9 @@
 import React from 'react'
 import './button.css';
 
-export const Button = (props) =>{
-     function handleClick(props) {
-        console.log('The link was clicked.');
-      }
+export const Button = ({text,handler}) =>{
```

Comment:
handler isn't correct name. Please rename it. For example onClick.  Because handler is very general name. You must give name to variables or props which will describe what they do.